tests/test_client.py

```diff
@@ -20,3 +22,30 @@
 def test_user_agent(user_agent, expected):
     client = AsyncClient(api_key='123', api_url='http:\\test', user_agent=user_agent)
     assert client.user_agent == expected
+
+
+@pytest.mark.asyncio
+@mock.patch("zyte_api.aio.client._post_func")
```

Comment:
As for the test, it could be nice to move the mockserver implementation from scrapy-zyte-api to python-zyte-api (see https://github.com/scrapy-plugins/scrapy-zyte-api/blob/137050522c14e146ed6bb28acc31dfde3fbdb6dd/tests/mockserver.py#L55); this would allow writing tests without using mocks. 

Please let me know if you want to give it a try, or if you prefer for it to be handled separately; I'm fine with merging without it.

Reply:
I think this won't be needed with the latest changes since it's simpler now and doesn't need mocks.